packages/larva-tokens/src/brands/rollingstone-2022.json

```diff
@@ -194,42 +195,64 @@
 		"name": "COLOR_GREY_LIGHT"
 	  },
 	  "FONT_FAMILY_BASIC": {
-		"category": "font-family",
-		"value": "'Graphik', sans-serif",
-		"type": "font-family",
-		"originalValue": "'Graphik', sans-serif",
-		"name": "FONT_FAMILY_BASIC"
-	  },
-	  "FONT_FAMILY_BODY": {
-		"category": "font-family",
-		"value": "'Georgia', Helvetica, sans-serif",
-		"type": "font",
-		"originalValue": "'Georgia', Helvetica, sans-serif",
-		"name": "FONT_FAMILY_BODY"
-	  },
-	  "FONT_FAMILY_PRIMARY": {
-		"category": "font-family",
-		"value": "'Publico Headline', serif",
-		"type": "font",
-		"comment": "Fallback for primary-fancy",
-		"originalValue": "'Publico Headline', serif",
-		"name": "FONT_FAMILY_PRIMARY"
-	  },
-	  "FONT_FAMILY_PRIMARY_FANCY": {
-		"category": "font-family",
-		"value": "'Publico Banner', serif",
-		"type": "font",
+        "category": "font-family",
+        "value": "sans-serif",
+        "type": "font-family",
+        "originalValue": "sans-serif",
+        "name": "FONT_FAMILY_BASIC"
+      },
+	  "FONT_FAMILY_BASIC_FANCY": {
+        "category": "font-family",
+        "value": "'Graphik Semibold', sans-serif",
+        "type": "font-family",
+        "originalValue": "'Graphik Semibold', sans-serif",
+        "name": "FONT_FAMILY_BASIC"
+      },
+      "FONT_FAMILY_BODY": {
+        "category": "font-family",
+        "value": "Helvetica, sans-serif",
+        "type": "font",
+        "originalValue": "Helvetica, sans-serif",
+        "name": "FONT_FAMILY_BODY"
+      },
+	  "FONT_FAMILY_BODY_FANCY": {
+        "category": "font-family",
+        "value": "'Georgia', Helvetica, sans-serif",
+        "type": "font",
+        "originalValue": "'Georgia', Helvetica, sans-serif",
+        "name": "FONT_FAMILY_BODY"
+      },
+      "FONT_FAMILY_PRIMARY": {
```

Comment:
@prajapatisagar The FONT_FAMILY_PRIMARY token should be a websafe fallback font - in this case, probably Georgia. 

For FONT_FAMILY_BODY, it is not using a webfont that needs to be loaded externally, so both the FONT_FAMILY_BODY and FONT_FAMILY_BODY_FANCY tokens can be the same.

The way you have FONT_FAMILY_SECONDARY is correct - you could specify a specific websafe typeface (see [a list here](https://www.w3schools.com/cssref/css_websafe_fonts.asp)), but it's also acceptable to only have serif or sans-serif.

Reply:
Also - this file will need to have the values added for font sizes, which you can add with the form, but I think it makes sense to wait until we have the Desktop XL breakpoint to do that.

See [this file for an example](https://github.com/penske-media-corp/pmc-larva/blob/master/packages/larva-tokens/src/brands/robbreport.json#L172) of what it looks like to have all of those added.

Reply:
@laras126 
The FONT_FAMILY_PRIMARY token should be a websafe fallback font - in this case, probably Georgia.
**Updated FONT_FAMILY_PRIMARY.**

For FONT_FAMILY_BODY, it is not using a webfont that needs to be loaded externally, so both the FONT_FAMILY_BODY and FONT_FAMILY_BODY_FANCY tokens can be the same.
**Changed value to Georgia**

The way you have FONT_FAMILY_SECONDARY is correct - you could specify a specific websafe typeface (see [a list here](https://www.w3schools.com/cssref/css_websafe_fonts.asp)), but it's also acceptable to only have serif or sans-serif.
**As per my knowledge here not needed changes.**

Also - this file will need to have the values added for font sizes, which you can add with the form, but I think it makes sense to wait until we have the Desktop XL breakpoint to do that.
**Yes right we will update it once breakpoint added.**